docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+# Use the default network olog_default.
+version: "3.9"
+services:
+  olog:
+    build: .
+    ports:
+      - "8181:8181"
+    depends_on:
+      - mongo
+      - elastic
+    # Ensure that the elastic server is up before starting olog.
+    command: ["wait-for-it", "-t", "30", "elastic:9200", "--", "java", "-jar", "target/olog-es-2.0.jar", "--spring.config.name=application-docker"]
+  mongo:
+    image: mongo
+    ports:
+      - 27017:27017
+    volumes:
+      - olog-mongodata:/etc/mongo
+  elastic:
+    image: elasticsearch:6.8.13
+    environment:
```

Comment:
Just a tip:
Environment variables can also be defined using dictionary format:
```
cluster.name: elasticsearch
bootstrap.memory_lock: 'false'
ES_JAVA_OPTS: '-Xms512m -Xmx512m'
```
I tend to find this easier to read, but it's of course just my personal opinion.